sys/riscv/boot.c

```diff
@@ -84,50 +87,24 @@
 
 /* NOTE: the boot stack is used before we switch out to `thread0`. */
 static alignas(STACK_ALIGN) uint8_t boot_stack[PAGESIZE];
-/*
- * Bare memory boot functions.
- */
-
-__boot_text static __noreturn void halt(void) {
-  for (;;)
-    __wfi();
-}
 
 /*
  * Bare memory boot data.
  */
 
-/* Last physical address used by kernel for boot memory allocation. */
-__boot_data static void *bootmem_brk;
-
-static __noreturn void riscv_boot(paddr_t dtb, paddr_t pde, paddr_t kern_end);
+static __noreturn void riscv_boot(void *dtb, paddr_t pde, paddr_t bootmem_end,
```

Comment:
Please move this declaration after the definition of the `_kerne_start ...` variables as the comment above says "Bare memory boot data".

Reply:
Done!